lua/macro-to-lua.md

```diff
@@ -14,7 +14,9 @@ Before getting started, its recommended to have [VS Code](https://code.visualstu
 
 ## Resources
 - [Programming in Lua](https://www.lua.org/pil/)
-- [Coldblooded's Lua autocompletion](https://gitlab.com/Coldblooded/mq-emmylua-definitions)
+- Coldblooded's Lua autocompletion. Requires [this](https://github.com/sumneko/lua-language-server) VSCode extension.
+    - [MQ definitions](https://github.com/johnfking/mq-definitions)
+    - [IMGUI definitions](https://github.com/johnfking/mq-imgui-definitions)

```

Comment:
I would prefer not to have this duplicated, it creates a maintenance issue if anything changes.  You can have it See Also for the reference to the other page which would get them to the same place.

Note this goes for both the PIL and the line here.